apps/checker/app/matchers/LanguageToolMatcher.scala

```diff
@@ -138,30 +137,30 @@ class LanguageToolFactory(
   }
 
   private def annotateExceptionWithRuleId(e: Throwable, ruleId: String, additionalMessage: String = "") =
-    new Exception(s"Error applying LanguageTool rule `${ruleId}`: ${e.getMessage} ${if (additionalMessage.size > 0) additionalMessage else ""}".trim, e)
+    new Exception(s"Error applying LanguageTool rule `${ruleId}`: ${e.getMessage} ${if (additionalMessage.nonEmpty) additionalMessage else ""}".trim, e)
 }
 
 object LanguageToolMatcher extends MatcherCompanion {
-  def getType = "languageTool"
+  def getType() = "languageTool"
```

Comment:
It feels like we may be a little inconsistent with brackets. [This post ](https://stackoverflow.com/questions/7600910/difference-between-function-with-parentheses-and-without)describes the different between empty parameters and parameterless. It seems like this would be an example of parameterless. Not very familiar with this distinction is apologies in advance if I've misunderstood.

Reply:
I probably did this to satisfy IntelliJ 😅 Happy to remove.

Reply:
Ah, yep, it was asking us to be consistent with a change that's been incorrect for a while – paren methods in the parent trait. Modified upstream.

Reply:
I'm leaving these changes for now, as they require lots of changes downstream from the fn definitions. Probably better applied globally to project via scalafmt or similar.